src/utils/requestManager.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
```

Comment:
Can we move this file to `helpers` folder as per our JS [convention](https://nimblehq.co/compass/development/code-conventions/javascript/#front-end-js)?
![image](https://user-images.githubusercontent.com/14927672/108680448-13fdd800-7518-11eb-88d1-00c6f8472db9.png)